public/data/SebasttianDaza.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "Sebastian Daza",
+  "type": "personal",
+  "bio": "Software Engineer, FullStack Developer and Open source enthusiast.",
+  "avatar": "https://avatars.githubusercontent.com/u/81209181?v=4",
```

This is a shorter way of using your image link. Plus, it updates automatically.
```suggestion
  "avatar": "https://github.com/SebasttianDaza.png",
```

---

Good to know, Thanks